lib/cc/engine/analyzers/helpers/main.rb

```diff
@@ -6,19 +6,30 @@ module Analyzers
       module Helpers
         BASE_POINTS = 10_000
 
+        def parsed_hashes
+          @parsed_hashes ||= []
+        end
+
         def flay
           @flay ||= ::Flay.new(flay_options)
         end
 
-        def new_violation(issue, location, other)
+        def report
+          flay.report(StringIO.new).each do |issue|
+            location = issue.locations.first
+            io.puts "#{new_violation(issue, location).to_json}\0"
```

Comment:
This turns out to be an include unfortunately. I think I'm going to make flay/reporting its own class and use it in the analyzers instead of including it.